.env.example

```diff
@@ -1,4 +1,10 @@
 CRAFT_ENDPOINT = "https://test-craft.library.ucla.edu/api"
 LIBCAL_ENDPOINT = "https://test.proxy.calendar.library.ucla.edu/"
 S3_BUCKET = "https://static.library.ucla.edu/"
+# LOCAL SITE USING nuxt generate and nuxt start commands
+SITEMAP_HOST="http://localhost:54359"
```

Comment:
If this is supposed to point to the nuxt site, it should probably use be port 3000. That's the default - 54359 looks like one of the random high ports it picks if 3000 is already taken.

Since it's just the example .env I'll still approve in case it's not worth changing

Reply:
Its static server when you run nuxt start after nuxt generate.